docs/control-structures.rst

```diff
@@ -337,8 +353,9 @@ arising when writing manual assembly by the following features:
 We now want to describe the inline assembly language in detail.
 
 .. warning::
-    Inline assembly is still a relatively new feature and might change if it does not prove useful,
-    so please try to keep up to date.
+    Inline assembly is a way to access the Ethereum Virtual Machine
+    at a low level. This discards and allows you to bypass several safety
```

Comment:
`discards and allows you bypass` is repetitive. Probably `This discards several important safety` is enough.